src/algebra/module/pi.lean

```diff
@@ -134,8 +134,8 @@ instance mul_action_with_zero' {g : I → Type*} [Π i, monoid_with_zero (g i)]
 { ..pi.mul_action',
   ..pi.smul_with_zero' }
 
-instance distrib_mul_action (α) {m : monoid α} {n : ∀ i, add_monoid $ f i}
-  [∀ i, distrib_mul_action α $ f i] :
+instance distrib_mul_action (α) {m : monoid α} {n : Π i, add_monoid $ f i}
+  [Π i, distrib_mul_action α $ f i] :
```

Comment:
This one is ok

Reply:
What is the rule, then?

Reply:
I just told you that in https://github.com/leanprover-community/mathlib/pull/9123#discussion_r706020757!